pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -71,8 +71,7 @@ import (
 const (
 	dockerProcessName     = "dockerd"
 	dockerPidFile         = "/var/run/docker.pid"
```

Comment:
Maybe add a comment here that this is the best effort lookup as pid file may be overridden: https://docs.docker.com/engine/reference/commandline/dockerd/#daemon:~:text=%2Dp%2C%20%2D%2Dpidfile%20string,for%20daemon%20PID%20file%20(default%20%22%2Fvar%2Frun%2Fdocker.pid%22)